indexer/services/roundtable/src/tasks/track-lag.ts

```diff
@@ -0,0 +1,90 @@
+import { axiosRequest, logger, stats } from '@dydxprotocol-indexer/base';
+import { BlockFromDatabase, BlockTable, IsoString } from '@dydxprotocol-indexer/postgres';
+import Big from 'big.js';
+import { DateTime } from 'luxon';
+
+import config from '../config';
+
+const VALIDATOR_BLOCK_HEIGHT_URL_SUFFIX = ':26657/block';
```

Comment:
Change this to `/block` and include `:26657` on the URL, this makes it so that we can use other full nodes that may map the Tendermint RPC to port 80 or another port as well.